inbox/quick-response.xml

```diff
@@ -0,0 +1,212 @@
+    <jid>zash@zash.se</jid>
+  </author>
+  <author>
+    <firstname>Tim</firstname>
+    <surname>Henkes</surname>
+    <email>me@syndace.dev</email>
+  </author>
+  <revision>
+    <version>0.1.0</version>
+    <date>2020-04-20</date>
+    <initials>th</initials>
+    <remark>
+      <ul>
+        <li>Renamed to better reflect the use-case</li>
+        <li>Split into two use-cases, one for textual responses and one for quick access to actions</li>
+        <li>Clarified i18n and general behavioral details</li>
+      </ul>
+    </remark>
+  </revision>
+  <revision>
+    <version>0.0.2</version>
+    <date>2018-09-30</date>
+    <initials>ka</initials>
+    <remark>Polishing preparing for submission</remark>
+  </revision>
+  <revision>
+    <version>0.0.1</version>
+    <date>2018-09-02</date>
+    <initials>ka</initials>
+    <remark>Initial version</remark>
+  </revision>
+</header>
+
+<section1 anchor="intro" topic="Introduction">
+  <p>Interactions with bots often require sending one of multiple predefined (plaintext) messages. This specification offers a way for XMPP entities to list the accepted responses to a message, so that entities that receive such a list can offer convenient UI to quickly respond with one of the allowed responses. Additionally, this specification provides a way for entities to provide generic actions in similar fashion to quick responses.</p>
+</section1>
+
+<section1 anchor="reqs" topic="Requirements">
+  <ul>
+    <li>Offer a simple way to list allowed responses to a message.</li>
+    <li>Offer a solution that doesn't rely on the receiving device to support &xepname;.</li>
+    <li>Allow internationalized responses.</li>
+  </ul>
+</section1>
+
+<section1 anchor="usecases" topic="Use Cases">
+  <p>A chat bot wants to provide a list of allowed responses to a message it sends.</p>
+  <p>A chat bot wants to provide quick access to certain actions for convenience.</p>
+</section1>
+
+<section1 anchor="elements" topic="Elements">
+  <section2 anchor="allowed-response-element" topic="Allowed Response">
+    <p>Each allowed response is represented by an <tt>&lt;allowed-response&gt;</tt> element in the <tt>&ns;</tt> namespace.</p>
+
+<example><![CDATA[<allowed-response xmlns="]]>&ns;<![CDATA[">
+  <translation xml:lang="en" value="yes" label="Yes!" />
+  <translation xml:lang="sv" value="ja" label="Ja!" />
+</allowed-response>]]></example>
+
+    <dl>
+      <di>
+        <dt>&lt;translation&gt;</dt>
+        <dd>One for each <tt>&lt;body&gt;</tt> and <tt>@xml:lang</tt> included in the <tt>&lt;message&gt;</tt> this response is a part of. The <tt>@value</tt> is the internationalized textual payload to put into the <tt>&lt;body&gt;</tt> of the message stanza that is sent when this response is selected. The <tt>@label</tt> is an optional internationalized textual label for this response. Clients that offer UI for quick selection of one of the allowed responses MAY refer to this response by label instead of value.</dd>
```

Comment:
I don't think the `@attr` syntax is something we normally use in XEPs (I've never seen it anyways). We probably shouldn't introduce new syntax in this XEP.

Reply:
Thank you so much for reviewing!

I'll only answer meta stuff, i.e. regarding formatting etc. I think discussion regarding the content of XEPs is generally discouraged in the git comments (right?).

The `@attr` was used (once :D) in Simple Buttons and I like how it clearly identifies the thing as an attribute, but if that's not used anywhere else I'll change it.

What do you mean by "preformatted text"? The `<tt>`s I'm wrapping them in?

Reply:
> What do you mean by "preformatted text"? The <tt>s I'm wrapping them in?

yah. I like how it looks, but I don't think it's used anywhere else and don't know if it will show up well printed or anything, so it's worth thinking about using quotes or something instead.

Reply:
I removed the `<tt>`s for elements, the `<>` already highlight elements quite well. For attribute I'm not sure, I think I'll just leave it at the `<tt>`s and see if someone else has thoughts on this.